src/js_schema.hpp

```diff
@@ -386,6 +396,9 @@ typename T::Object Schema<T>::object_for_property(ContextType ctx, const Propert
     static const String optional_string = "optional";
     Object::set_property(ctx, object, optional_string, Value::from_boolean(ctx, is_nullable(property.type)));
 
+    static const String mapped_to_string =  "mappedTo";
```

Comment:
Right now I use `mapTo` when defining the schema, but `mappedTo` is returned when calling `realm.schema`. Should these be the same?

Reply:
Yes, I think so.